src/odb/include/odb/db.h

```diff
@@ -425,7 +420,7 @@ class dbDatabase : public dbObject
   ///
   /// Create an instance of a database
   ///
-  static dbDatabase* create();
+  static dbDatabase* create(utl::Logger* logger = nullptr);
```

Comment:
When would we want to allow a nullptr?

Reply:
we would not. In the function implementation, I create a new logger if none is passed.